docs/concepts/configuration/overview.md

```diff
@@ -119,4 +89,4 @@ This document is meant to highlight and consolidate in one place configuration b
 
 - Use `kubectl run` and `expose` to quickly create and expose single container Deployments. See the [quick start guide](/docs/user-guide/quick-start/) for an example.
 
-
+{% endcapture %}
```

Comment:
You will need to add the template include tag at the end of the file:

{% include templates/concept.md %}